engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/db/PeriodicProcessesRepository.scala

```diff
@@ -48,17 +49,36 @@ object PeriodicProcessesRepository {
     )
   }
 
-  def createPeriodicProcess(processEntity: PeriodicProcessEntity): PeriodicProcess = {
+  def createPeriodicProcess(processEntity: PeriodicProcessEntity): PeriodicProcess[CanonicalProcess] = {
     val processVersion = createProcessVersion(processEntity)
     val scheduleProperty = decode[ScheduleProperty](processEntity.scheduleProperty)
       .fold(e => throw new IllegalArgumentException(e), identity)
     PeriodicProcess(
       processEntity.id,
       model.DeploymentWithJarData(
         processVersion = processVersion,
-        canonicalProcess = processEntity.processJson,
         inputConfigDuringExecutionJson = processEntity.inputConfigDuringExecutionJson,
-        jarFileName = processEntity.jarFileName
+        jarFileName = processEntity.jarFileName,
+        process = processEntity.processJson.get // todo: add exception ?
```

Comment:
Yes, please , with some basic information from context, so that if it ever fails we'll have at least something useful in logs